### ISSUE TITLE
You must call one of in() or append() methods before iterating over a Finder.  

### ISSUE THREAD

**Issue by jozefrebjak** [codeigniter4/devkit#102]
If we copy php-cs-fixer config from templates and run php-cs-fixer command, then we get error within run:

```sh
In Finder.php line 661:
                                                                                 
  [LogicException]                                                               
  You must call one of in() or append() methods before iterating over a Finder.  
```

**Comment by kenjis** [comment]
Cannot reproduce. `in()` is called: https://github.com/codeigniter4/devkit/blob/a7dc5842932c29ef21fc182e00a6209acb3ceed4/src/Template/.php-cs-fixer.dist.php#L9

```
$ php-cs-fixer fix --ansi --verbose --diff
PHP CS Fixer 3.22.0 Chips & Pizza by Fabien Potencier and Dariusz Ruminski.
PHP runtime: 8.1.21
Loaded config CodeIgniter4 Coding Standards from "/Users/kenji/work/codeigniter/tmp/ci436/.php-cs-fixer.dist.php".
Using cache file "build/.php-cs-fixer.cache".
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
Legend: .-no changes, F-fixed, S-skipped (cached or empty file), I-invalid file syntax (file ignored), E-error

Fixed 0 of 52 files in 0.017 seconds, 16.000 MB memory used
```

**Comment by kenjis** [comment]
If you still have this issue, feel free to reopen.
